q.js

```
@@ -2022,6 +2032,10 @@ Promise.prototype.nodeify = function (nodeback) {
     }
 };
 
+Q.noConflict = function() {
+    console.log('Q.noConflict only works when Q is used as a global');
```

This should simply throw an error.